features/0755-oca-for-aries/README.md

```diff
@@ -0,0 +1,402 @@
+label to be used for each attribute for a given language. The label overlay also includes labels for
+attributes with enumerated values. For example, if a data attribute might contains the codes "EN", "FR", "SP"
+that mean "English", "French" and "Spanish", respectively.
+- the multilingual [information
+overlay](https://oca.colossi.network/v1.1.0-rc.html#information-overlay) defines
+a description or help text about each attribute for a given language. There will
+be one overlay per issuer-supported language.
+- the multilingual [meta overlay](https://oca.colossi.network/v1.1.0-rc.html#meta-overlay) contains
+information about the credential itself, such as "name" and "description."
+
+##### The AnonCreds "dateint" Format
+
+A common data format used in Aries AnonCreds verifiable credentials is "`dateint`", as
+described in [RFC 0592](../0592-indy-attachments/README.md). A `dateint` is a date stored
+as an integer (for example `September 29, 2022` is the integer `20,220,929`). In the format
+overlay, `dateint` is a permitted value when used with Aries. An recipient of an OCA Bundle
+with a `dateint` SHOULD extract from the integer data value the year, month and day and
+display those elements in the style appropriate for the user. For example, a mobile app
+should display the data as a date based on the user's language/country setting.
+
+#### Aries Specific Overlays
+
+In addition to the core OCA Overlays listed above, Aries agents also use an
+additional "extension" overlay, the [credential_branding overlay](#), that
+provides the issuer with a way to provide a series of data elements about the
+credential branding that they would like to see applied. The credential_branding
+overlay provides a list of name/value pairs, and the holders (and verifiers) can
+use the values when displaying the credential of that type from that issuer. The
+holder/verifier controls the details of how the credentials are displayed, but
+the issuer does have some input into the branding. The following are the branding
+data elements the issuer can set:
+
+- logo_uri - a URI pointing to a logo to display with the credential.
+- logo_base64 - an inline, base64-encoded logo to display with the credential.
+- background_image_uri - a URI pointing to a background image to display under
+  the credential data.
+- background_image_base64 -  - an inline base64-encoded background image to display under the
+  credential data.
+- header_background_color - an RGB color code for the credential display header
+  background color.
+- footer_background_color - an RGB color code for the credential display footer
+  background color.
+- header_font_color - an RGB color code for the credential display header font
+  color.
+- footer_font_color - an RGB color code for the credential display footer font
+  color.
+- primary_attribute - the name of the primary capture base attribute to display
```

Comment:
what is the "primary attribute" and "secondary_attribute"? is that the "Issuer Name" and "Credential Name", respectively?

Reply:
The idea is to allow the issuer to suggest to the wallet (display software) the "best" attributes from the credential to display on top of the background image.  In other words, if the wallet (or whatever) displays the "card", it could follow the issuers suggestion and put one or both of the primary and secondary values (perhaps with their multilingual labels).  They might put those instead of "Issue Date" (which is NOT an attribute in the credential -- it's metadata collected by the wallet), or in addition to "Issue Date".  Without some guidance from the issuer, the wallet would have no information to decide which attributes might be helpful to show.

Does that sound useful?  I confess that it was an idea of mine that I thought might be useful.